api/docs/release.dox

```diff
@@ -128,6 +128,14 @@ The changes between version \DR_VERSION and 11.0.0 include the following compati
 changes:
  - Added X64 Linux support to dr_create_memory_dump(). This API has the same
```

Comment:
This one is in the wrong list: please move it to the non-compatibility-affecting changes list below on line 133.